packages/core/src/repositories/runErrors/documentLogsRepository/index.ts

```diff
@@ -38,4 +42,19 @@ export class DocumentLogsWithErrorsRepository extends Repository<
       .where(eq(workspaces.id, this.workspaceId))
       .as('documentLogsWithErrorsScope')
   }
+
+  async findByUuid(uuid: string) {
```

Comment:
check there's an index for this column in the model, i think there wasn't

Reply:
![image](https://github.com/user-attachments/assets/e6855aa7-939d-43e0-8a89-17d6ed6e950c)

Reply:
This has being like this since forever I think. Feel free to add it. I'll add a FIXME not doing here for sure 👍